packages/terra-modal-manager/src/ModalManager.jsx

```diff
@@ -190,20 +192,19 @@ class ModalManager extends React.Component {
 
   render() {
     const { closeModal, size, isFocused, isOpen, isMaximized } = this.props;
-
-    const modalClassNames = classNames([
-      'terra-ModalManager-modal',
-      { [`terra-ModalManager-modal--${size}`]: !(isMaximized || this.forceFullscreenModal) },
+    const isFullscreen = isMaximized || this.forceFullscreenModal;
+    const modalClasses = cx([
```

Comment:
Unfortunately, the Modal doesn't expose custom attributes, just classes. The Modal itself adds its own '.modal' class.